docs/error-reporting-usage.rst

```diff
@@ -0,0 +1,46 @@
+Using the API
+=============
+
+
+Authentication and Configuration
+--------------------------------
+
+- For an overview of authentication in ``gcloud-python``,
+  see :doc:`gcloud-auth`.
+
+- In addition to any authentication configuration, you should also set the
+  :envvar:`GCLOUD_PROJECT` environment variable for the project you'd like
+  to interact with. If you are Google App Engine or Google Compute Engine
+  this will be detected automatically.
+
+- After configuring your environment, create a
+  :class:`Client <gcloud.logging.client.Client>`
+
+  .. doctest::
+
+     >>> from gcloud import error_reporting
+     >>> client = error_reporting.Client()
+
+  or pass in ``credentials`` and ``project`` explicitly
+
+  .. doctest::
+
+     >>> from gcloud import error_reporting
+     >>> client = error_reporting.Client(project='my-project', credentials=creds)
+
+
+Reporting an exception
+-----------------------
+
+Report a stacktrace to Stackdriver Error Reporting after an exception
+
+.. doctest::
+
+   >>> from gcloud import error_reporting
+   >>> from
+   >>> client = error_reporting.Client()
+   >>> try:
+   >>>     raise NameError
+   >>> except Exception:
+   >>>     client.report_error(message="Something went wrong")
```

Comment:
Also, consider renaming report_error() to report(). This is how the method is named in Node.js